cfgov/unprocessed/apps/owning-a-home/css/explore-rates.less

```diff
@@ -437,7 +437,7 @@
     position: absolute;
     z-index: 1000;
     right: 0;
-    top: 0;
+    top: -26px;
```

Comment:
This gets really tight on narrower screens

<img width="423" alt="Screenshot 2023-09-26 at 8 20 38 AM" src="https://github.com/cfpb/consumerfinance.gov/assets/704760/12475c49-3c0a-4fe3-b6fd-2e0bde404cf0">

Instead of negative positioning the menu, what do you think about bumping the `<figure>` below it (the actual chart) down 25px instead? See below

<img width="797" alt="Screenshot 2023-09-26 at 8 41 29 AM" src="https://github.com/cfpb/consumerfinance.gov/assets/704760/bd1fa738-c288-43f6-8d84-42cf751d8bb0">

Reply:
(Above screenshot is 25px padding-top on the figure)

Reply:
I agree it's tight at that range, the problem is though (at mobile sizes especially) the hover/popup is still clipped by the button overlay:
<img width="508" alt="Screenshot 2023-09-26 at 8 56 47 AM" src="https://github.com/cfpb/consumerfinance.gov/assets/1558033/36ae1911-58e0-4f15-9d7d-64dabbaafe63">

Best thing to do (imo) is to move it up into the gap between the h2 and the top rule of the chart and have it a bit ugly for tablet sizes.

Reply:
Hmmmm won't this be an issue anyway when the chart download dialog is expanded? Also, shouldn't it have a top border when it is expanded?

What about this?

- Bump the chart down with `padding-top: 25px` on the `<figure>`
- Remove `z-index: 0` from `highcharts-container` class

Fiddling in the browser, that appears to negate the stacking issue, with the tooltip then appearing above the dropdown. Although, maybe that shouldn't apply when the menu is open, so maybe only remove the z-index from the container when `chart-menu__closed` is present.

---

Alternatively, could we move this menu to below lower-right area of the chart so it don't run into things based on their height?

Reply:
I'd prefer not to bump down the chart just to make better use of vertical space. Could we move the download to the lower right area of the chart? yeah, but I'd prefer limiting the changes where possible and this preserves the original design more or less while removing the janky bits.

Reply:
I don't want to hold this up if you need it in, but doesn't this not meet our spacing standards? Maybe have design weigh in, if you haven't. The chart has a line at the top and a line at the bottom and then the chart lines in the middle. The chart lines look to have a 150px gap between them, and then the gap between the top line and the top of the chart looks like 100px, and the bottom of the chart line to the bottom line is about 210px. It seems like the gap between the top of the chart lines and the top line of the chart should either be much tighter, or should be the width of the chart lines or more, and that any content that moves above that line should have at least 10px of space after the content above it. Also, the download button still needs a top border if it moves off the line, yea?